apollo-router/tests/snapshots/integration_tests__validation_errors_from_rust.snap

```diff
@@ -4,23 +4,17 @@ expression: response.errors
 ---
 [
   {
-    "message": "cannot query field `inStock` on type `Product`",
-    "locations": [
-      {
-        "line": 4,
-        "column": 5
-      }
-    ],
+    "message": "compiler error: fragment `Unused` must be used in an operation",
     "extensions": {
       "code": "GRAPHQL_VALIDATION_FAILED"
     }
   },
   {
-    "message": "cannot query field `notAField` on type `Query`",
+    "message": "compiler error: the argument `notAnArg` is not supported",
     "locations": [
       {
-        "line": 6,
-        "column": 3
+        "line": 1,
+        "column": 22
```

Comment:
I thought the locations refered to the graphql operation, if so why do the lines and columns change?

Reply:
The input changed to have a different error: https://github.com/apollographql/router/pull/4038#discussion_r1365851704
The new error has a different message and location

Reply:
thanks!